deployments/liqo_chart/subcharts/liqodash_chart/values.yaml

```diff
@@ -4,7 +4,7 @@
 
 image:
   repository: liqo/dashboard
-  pullPolicy: IfNotPresent
+  pullPolicy: Always
```

Comment:
Is there any reason we do always "pull"?

Reply:
I guess not, it was just some testing leftover. Thank you for pointing that out!